piker/pp.py

```diff
@@ -957,7 +981,8 @@ def open_pps(
             expiry = pendulum.parse(expiry)
 
         pp = pp_objs[bsuid] = Position(
-            Symbol.from_fqsn(fqsn, info={}),
+            Symbol.from_fqsn(
+                fqsn, entry['symbol']),
```

Comment:
As mentioned in chat, we need to do a `entry.get('symbol')` here and instead just fill in a default `Symbol` here to continue supporting old `pps.toml` files.

Further for entries detected as `crypto` we should probably put a big warning that the entry needs to be recalculated to avoid the precision error this patch was originally meant to fix!